File: ruby/001-two-sum.rb
# @param {Integer[]} nums
# @param {Integer} target
# @return {Integer[]}
def two_sum(nums, target)
    nums.each_with_index do |num, index|
        ((index + 1)...nums.length).each do |ix|
            return [index, ix] if (num + nums[ix] == target)
        end
    end
    return []
end

File: python/001-two-sum.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        # Brute Force
        """
        [Two Sum Problem](https://leetcode.com/problems/two-sum/)

        Args:
            nums (List[int]): List of integers.
            target (int): Integer.

        Returns:
            List[int]: List of intgers which sum up to target.
        """
        for i in range(0, (len(nums))):
            for j in range(i + 1, (len(nums))):
                if nums[i] + nums[j] == target:
                    return [i, j]
